homeworks/anatoliy.pogoriliy_AyzekUorren/homework_4/task_2.js

```diff
@@ -0,0 +1,26 @@
+function uploadProgress(obj, urlFile) {
```

Comment:
Accepted. But the idea was to have third argument of function as callback for upload progress event, so far you are just logging the progress but it could be anything like reflecting loading progress in state, send messages etc. 

So expectation was smth like `function uploadProgress(obj, urlFile, progressCallback) {...`

Anyway, you have got the idea

Reply:
Yes, i thought so.
Sorry but forgot to add arg.